### ISSUE TITLE
Add peak_load value to LVLoad objects

### ISSUE THREAD

**Issue by gplssm** [openego/dingo#158]
Currently, LV loads don't include the value of its actual peak load.

- [x] Add argument to class
- [x] Add load values for LV strings of retail/industrial and agricultural
- [x] Add load values for LV strings of residential

**Comment by gplssm** [comment]
3 times is better than only once: duplicate of #77 #128. Progress documentation and discussion takes place here.
